### DISCUSSION TITLE
Starting and stopping a log entry

### DISCUSSION THREAD

**Discussion by amos-kibet** [3Stones-io/omedis#270] [General]
We realised we need a well defined and a user-friendly interface that enables starting and stopping a log entry. This will fix #271.
@fradarr, @almirsarajcic, @wintermeyer, we can continue the discussion here.

**Comment by fradarr** [comment]
Since the TimeTracking is central in our App it should have a central place. Whatever you do in the app you should be able to see where your time is been "counted in" so I would propose to have a new region in the search bar like this: 

<img width="1312" alt="Screenshot 2024-11-11 at 07 03 14" src="https://github.com/user-attachments/assets/d670c7d1-e942-4c71-8da9-aa3308b6cfab">

**Comment by wintermeyer** [comment]
Great idea!

**Comment by wintermeyer** [comment]
Do we need the search bar at all? Right now we don't have a search. And we can do the search (if needed) somewhere else. When TimeTracking is the essential part of the application we should follow @fradarr idea to make it the hero of the GUI. Right now the search gets prime estate but is not used at all.

**Reply by fradarr** [reply]
I would keep the search for now, when we think of having few hunderts of patients, documents, forms and so on .... it could be handy.

**Reply by wintermeyer** [reply]
> I would keep the search for now, when we think of having few hunderts of patients, documents, forms and so on .... it could be handy.

We could make the search accessible with a 🔎 symbol which opens a text input field. Right now we always have a text input field which takes quite a bit of space.

**Comment by fradarr** [comment]
When we push the idea further it (the red square) should be a menu where clicking on it gives you the "Activities" and you can easily pick one.
